contributor-ladder.md

```diff
@@ -0,0 +1,222 @@
+    * [Reviewer](###reviewer)
+    * [Maintainer](###maintainer)
+* [Inactivity](##inactivity)
+* [Involuntary Removal](##involuntary-removal)
+* [Stepping Down/Emeritus Process](##stepping-downemeritus-process)
+* [Contact](##contact)
+
+
+## Contributor Ladder
+
+Hello! We are excited that you want to learn more about our project contributor ladder! This contributor ladder outlines the different contributor roles within the project, along with the responsibilites and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows.  Our project members are happy to help you advance along the contributor ladder.
+
+Each of the contributor roles below is organized into lists of three types of things. "Responsibilities" are things that contributor is expected to do. "Requirements" are qualifications a person needs to meet to be in that role, and "Rights" are things contributors on that level are entitled to.
+
+
+### Community Participant
+<!--This role spells out what's expected of general community participants.  This is the lowest
+level of participation in the project, and as such many projects don't bother to
+define it and leave this section out.-->
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: A Community Participant participates in the community and contributes their time, thoughts, etc. Community participants are usually users who have stopped being anonymous and started being active in project discussions.
+
+* Responsibilities:
+    * Must follow the CNCF CoC
+* How users can get involved with the community:
+        * Participating in community discussions
+        * Helping other users
+        * Submitting bug reports
+        * Commenting on issues
+        * Trying out new releases
+        * Attending community events
+        * [TODO: Other examples of non-contribution participation]
+
+
+### Contributor
+<!-- This role describes people who have just started contributing, or who contribute occasionally but don't participate in project governance or have defined responsibilities.  Usually projects define either this level or Community Participant, but not both.  If you don't definte this role, make sure to copy over its requirements to Organization Member -->
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: A Contributor contributes directly to the project and adds value to it. Contributions need not be code. People at the Contributor level may be new contributors, or they may only contribute occasionally.
+
+* Responsibilities include:
+    * Following the CNCF CoC
+    * Following the project contributing guide
+* Requirements (one or several of the below):
+    * Reports and sometimes resolves issues
+    * Occasionally submits PRs
+    * Contributes to the documentation
+    * Regularly shows up at meetings, takes notes
```

Comment:
```suggestion
    * Regularly or semi-regularly shows up at meetings, takes notes
```

Reply:
I'm just going to remove the "regularly"